content/en/docs/pipelines/installation/overview.md

```diff
@@ -92,10 +151,27 @@ Interfaces
   * Kubeflow Pipelines UI via the **Open Pipelines Dashboard** link in the 
     GCP Console
   * Kubeflow Pipelines SDK in Cloud Notebooks
-
+  * Kubeflow Pipelines endpoint of your instance is auto-configured for you.
+
+Release Schedule
+: AI Platform Pipelines is available for a chosen set of stable Kubeflow
+Pipelines releases. You will receive updates slightly slower than Kubeflow
+Pipelines Standalone.
+
+Upgrade Support (**Alpha**)
+: In place upgrade is not supported. Reinstall new version while retaining data
+is supported without official documentation. You can
+[delete a AI Platform Pipelines application](https://cloud.google.com/ai-platform/pipelines/docs/getting-started#clean_up)
+**WITHOUT** selecting **Delete cluster** checkbox and reinstall the newer version
+in the same cluster and the same namespace. Persisted artifact and database data
```

Comment:
oops, missed this earlier.  This turns out to be NOT true for the MKP installation, only the 'standalone' installation.

Reply:
let's follow up in the issue, Renmin and I've both verified this works for marketplace -> marketplace upgrade.
There's probably sth unclear with the instructions.